ui/tests/acceptance/access/namespaces/index-test.js

```diff
@@ -3,24 +3,23 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { currentRouteName } from '@ember/test-helpers';
+import { currentRouteName, visit } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
 import { setupMirage } from 'ember-cli-mirage/test-support';
-import page from 'vault/tests/pages/access/namespaces/index';
-import authPage from 'vault/tests/pages/auth';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
 
 module('Acceptance | Enterprise | /access/namespaces', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
-  hooks.beforeEach(function () {
-    return authPage.login();
+  hooks.beforeEach(async function () {
+    return await login();
```

Comment:
these changes shouldn't be necessary 🤔 was this a linting thing?

Reply:
Nope, this was me trying to figure out why the enterprise tests were failing for seemingly _no reason_. It seemed to resolve the test failures initially, but because the failures were unclear - the UI just seemed to be stuck on unseal - I'm not sure if this helped in the end. 

I kept it because I liked removing page object because I strongly dislike it 😆 but I can revert this change if that's preferred! I'll have to double check the enterprise tests again